Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -485,300 +407,378 @@ describe(
+            expect(rgba[2]).toEqual(0);
+            expect(rgba[3]).toEqual(255);
+          });
         });
       });
     });
 
-    it("gets copyrights from gltf", function () {
-      const resource = Resource.createIfNeeded(boxWithCreditsUrl);
+    it("renders model with double-sided material", function () {
+      const resource = Resource.createIfNeeded(twoSidedPlaneUrl);
       return resource.fetchJson().then(function (gltf) {
         return loadAndZoomToModelExperimental(
           {
             gltf: gltf,
-            basePath: boxWithCreditsUrl,
+            basePath: twoSidedPlaneUrl,
+            modelMatrix: Transforms.eastNorthUpToFixedFrame(
+              Cartesian3.fromDegrees(0.0, 0.0, 100.0)
+            ),
           },
           scene
         ).then(function (model) {
-          const expectedCredits = [
-            "First Source",
-            "Second Source",
-            "Third Source",
-          ];
+          const renderOptions = {
+            scene: scene,
+            time: defaultDate,
+          };
 
-          scene.renderForSpecs();
-          const creditDisplay = scene.frameState.creditDisplay;
-          const credits =
-            creditDisplay._currentFrameCredits.lightboxCredits.values;
-          const length = credits.length;
-          expect(length).toEqual(expectedCredits.length);
-          for (let i = 0; i < length; i++) {
-            expect(credits[i].credit.html).toEqual(expectedCredits[i]);
-          }
+          const center = model.boundingSphere.center;
+          const range = 4.0 * model.boundingSphere.radius;
+          scene.camera.lookAt(
+            center,
+            new HeadingPitchRange(0, -CesiumMath.PI_OVER_TWO, range)
+          );
+
+          // The top of the double-sided plane should render brightly, since
+          // its normal is pointing towards the light
+          let result;
+          expect(renderOptions).toRenderAndCall(function (rgba) {
+            expect(rgba).not.toEqual([0, 0, 0, 255]);
+            result = rgba;
+          });
+
+          scene.camera.lookAt(
+            center,
+            new HeadingPitchRange(0, CesiumMath.PI_OVER_TWO, range)
+          );
+
+          // The bottom of the plane should render darker than the top, since
+          // its normal is pointing away from the light.
+          expect(renderOptions).toRenderAndCall(function (rgba) {
+            expect(rgba).not.toEqual([0, 0, 0, 255]);
+
+            expect(rgba[0]).toBeLessThan(result[0]);
+            expect(rgba[1]).toBeLessThan(result[1]);
+            expect(rgba[2]).toBeLessThan(result[2]);
+            expect(rgba[3]).toEqual(result[3]);
+          });
         });
       });
     });
 
-    it("displays all types of credits", function () {
-      const resource = Resource.createIfNeeded(boxWithCreditsUrl);
+    // This test works for Model but does not work for ModelExperimental.
+    xit("renders model with emissive texture", function () {
```

Comment:
I see what's happening here, this box has no normals but has an emissive texture.

Without normals, our implementation sets the lighting to `UNLIT` mode (which doesn't support emissive), but in this particular case, it really isn't an unlit, it's a PBR material without 

Not sure what to do about this in the short term, ideally we'd have #6506 so it would have normals and then it would go down the PBR path correctly...